structlog_sentry/__init__.py

```diff
@@ -168,6 +168,21 @@ def _handle_breadcrumb(self, event_dict: EventDict) -> None:
             event, hint = self._get_breadcrumb_and_hint(event_dict)
             self._get_hub().add_breadcrumb(event, hint=hint)
 
+    @staticmethod
+    def _get_level_value(level_name: str) -> int:
+        """Get numeric value for the log level name given."""
+        try:
+            # Try to get one of predefined log levels
+            return getattr(logging, level_name)
+        except AttributeError:
+            # May be it is a custom log level?
+            level = logging.getLevelName(level_name)
+            if isinstance(level, int):
+                return level
+
+            # Re-raise original error
+            raise
```

Comment:
I'm thinking that maybe this would be more understandable?

```suggestion
            raise ValueError(f"{level_name} is not a valid log level") from e
```